### PR TITLE
[NUI.Components] Open public apis of Components for ACR

### PR DESCRIPTION
**Pull request by junqingma**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/EFLComponentBasedApplication.cs [hunk 1]
```diff
@@ -11,7 +11,7 @@ namespace Tizen.Applications.ComponentBased.Default
     /// <since_tizen> 6 </since_tizen>
     public class EFLComponentBasedApplication : ComponentBasedApplication
     {
-        private const string LogTag = "Tizen.Applications.ComponentBasedApplication";
+        private const string LogTag = "Tizen.Applications";
```
Comment (dongsug-song) [inline src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/EFLComponentBasedApplication.cs]: Hello,
Why Application's code is changed? I think rebase or patch modify is required.
please check.
thank you.

File: src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/EFLWindowInfo.cs [hunk 2]
```diff
@@ -10,7 +10,7 @@ namespace Tizen.Applications.ComponentBased.Default
     /// <since_tizen> 6 </since_tizen>
     public class EFLWindowInfo : IWindowInfo
     {
-        private const string LogTag = "Tizen.Applications.EFLWindow";
+        private const string LogTag = "Tizen.Applications";
         private Window _win;
```
Comment (dongsug-song) [inline src/Tizen.Applications.ComponentBased.Default/Tizen.Applications.ComponentBased.Default/EFLWindowInfo.cs]: same as above

File: src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/ComponentBasedApplication.cs [hunk 3]
```diff
@@ -25,7 +25,7 @@ namespace Tizen.Applications.ComponentBased.Common
     /// <since_tizen> 6 </since_tizen>
     public abstract class ComponentBasedApplication : Application
     {
-        private const string LogTag = "Tizen.Applications.ComponentBasedApplicationBase";
+        private const string LogTag = "Tizen.Applications";
         private Dictionary<Type, ComponentStateManger> _componentFactories = new Dictionary<Type, ComponentStateManger>();
```
Comment (dongsug-song) [inline src/Tizen.Applications.ComponentBased/Tizen.Applications.ComponentBased.Common/ComponentBasedApplication.cs]: same as above

File: src/Tizen.NUI.Components/Controls/SelectButton.cs [hunk 4]
```diff
@@ -208,16 +208,16 @@ public FloatSelector CheckImageOpacitySelector
         /// <since_tizen> 6 </since_tizen>
         /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Size2D CheckImageSize2D
+        public Size CheckImageSize
```
Comment (agnelovaz) [inline src/Tizen.NUI.Components/Controls/SelectButton.cs]: What does the "Check" prefix mean?  Should update in the comments what the CheckImage is; an icon, background, selected or unselected image.
Should this have been renamed when the button got renamed to a SelectButton?

Comment (agnelovaz) [reply src/Tizen.NUI.Components/Controls/SelectButton.cs]: Checking at previous emails did we not want this to be the "icon", looks like the "Check" was not updated in all files when the button was renamed to SelectButton

**Comment by dongsug-song** [comment]
this is done by https://github.com/Samsung/TizenFX/pull/1061
so I am closing this.

**Pull request closed without merging**